x/storage/keeper/keeper.go

```diff
@@ -792,13 +851,29 @@ func (k Keeper) SealObject(
 
 	objectInfo.ObjectStatus = types.OBJECT_STATUS_SEALED
 
-	store := ctx.KVStore(k.storeKey)
 	bbz := k.cdc.MustMarshal(bucketInfo)
 	store.Set(types.GetBucketByIDKey(bucketInfo.Id), bbz)
 
 	obz := k.cdc.MustMarshal(objectInfo)
 	store.Set(types.GetObjectByIDKey(objectInfo.Id), obz)
 
+	if isUpdate {
+		if err := ctx.EventManager().EmitTypedEvents(&types.EventUpdateObjectContentSuccess{
+			Operator:        spSealAcc.String(),
+			BucketName:      bucketInfo.BucketName,
+			ObjectName:      objectInfo.ObjectName,
+			ObjectId:        objectInfo.Id,
+			ContentType:     objectInfo.ContentType,
+			PrevPayloadSize: prevPayloadSize,
+			NewPayloadSize:  objectInfo.PayloadSize,
+			PrevChecksums:   prevCheckSums,
+			NewChecksums:    objectInfo.Checksums,
+			Version:         objectInfo.Version,
+			UpdatedAt:       objectInfo.UpdatedAt,
+		}); err != nil {
+			return err
+		}
+	}
```

Comment:
we emit `EventUpdateObjectContentSuccess` and `EventSealObject` with `ForUpdate`, is it a bit redundant?

Reply:
removed `ForUpdate` as no longer needed